index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
+  var babelPlugins = pluginWrappers.map(function(wrapper) {
+    return wrapper.plugin;
```

Comment:
Should the property under which the transformer is looked up be changed to `transformer`? I copied this from the `ember-cli-htmlbars` addon, where AST plugins are added like this ...